android-base.json

```diff
@@ -0,0 +1,77 @@
+{
+  "extends": [
+    "config:base",
+    ":disableDependencyDashboard"
+  ],
+  "github-actions": {
+    "enabled": false
+  },
+  "packageRules": [
+    {
+      "groupName": "org.jetbrains.kotlin.*",
+      "matchPackagePrefixes": [
+        "org.jetbrains.kotlin:",
+        "org.jetbrains.kotlin."
+      ]
+    },
+    {
+      "groupName": "org.jetbrains.kotlinx.*",
+      "matchPackagePrefixes": [
+        "org.jetbrains.kotlinx:",
+        "org.jetbrains.kotlinx."
+      ]
+    },
```

Comment:
Curious about this. 😊 `org.jetbrains.kotlinx` don't usually go hand-in-hand, do they?

Reply:
They don't, it's just to have them in a single PR. :)

Reply:
Ok! I've been using this feature when updates follow logical groups, hence the confusion. 😄

Reply:
Yeah it's not a strictly logical group. We were trying to cut down on the number of PRs (without going all the way to a single PR). 😅